forge-gui/res/cardsfolder/upcoming/emperor_apatzec_intli_iv.txt

```diff
@@ -0,0 +1,12 @@
+Name:Emperor Apatzec Intli IV
+ManaCost:R G W
+Types:Legendary Creature Human Noble
+PT:3/4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Creature.Other+powerGE4 | TriggerZones$ Battlefield | Execute$ TrigPump | TriggerDescription$ Whenever another creature enters the battlefield under your control, that creature perpetually gains haste if its power is 4 or greater. If its toughness is 4 or greater, you gain 4 life. If its mana value is 4 or greater, seek a creature card.
```

Comment:
bad valid

Reply:
So it should be a single trigger with three conditional subabilities ?

Reply:
indeed